js-miniapp-sdk/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ### v1.1.0 (2020-7-21)
 
-- Added support for requesting geolocation permission from the host application to allow fetching of the coordinates data thereafter.
+- Added support for requesting geolocation permission from the host application to allow fetching of the coordinates data thereafter. [See here](https://github.com/rakutentech/js-miniapp/tree/master/js-miniapp-sdk#3-request-permissions)
```

Comment:
I think this should point to `README.md#3-request-permissions` instead, right?

Reply:
Done